tests/e2e_test.go

```diff
@@ -282,6 +305,160 @@ func TestSignedEntryTimestamp(t *testing.T) {
 	}
 }
 
+func setup(t *testing.T) {
+	if err := os.MkdirAll("testdata", 0755); err != nil {
+		t.Fatalf("Failed to create testdata directory: %v", err)
+	}
+
+	if _, err := os.Stat(testCAPath); os.IsNotExist(err) {
```

Comment:
We should compute the path to the test artifacts using `t.TempDir()` so they're cleaned up after test execution.